modules/fbx/doc_classes/FBXState.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="FBXState" inherits="GLTFState" experimental="" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+	</brief_description>
+	<description>
+		The FBXState handles the state data imported from FBX files.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="allow_geometry_helper_nodes" type="PackedInt32Array" setter="set_allow_geometry_helper_nodes" getter="get_allow_geometry_helper_nodes" default="PackedInt32Array()">
+		</member>
```

Comment:
It's very unlikely someone skimming through will know what this is about. May be worth documenting now.